bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -189,37 +140,22 @@ <h3>Hubs</h3>
     <li>hubs.mozilla.com</li>
     <li>reticulum.io</li>
   </ul>
-  
+
   <h3>Localization</h3>
   <ul class="mzp-u-list-styled">
-    <li>l10n.mozilla.org</li>
     <li>pontoon.mozilla.org</li>
   </ul>
 
   <h3>Observatory</h3>
   <ul class="mzp-u-list-styled">
-    <li>http-observatory.security.mozilla.org</li>
     <li>observatory.mozilla.org</li>
     <li>tls-observatory.services.mozilla.com</li>
   </ul>
 
-  <h3>Payment Subscription</h3>
-  <ul class="mzp-u-list-styled">
-    <li>prod.fxa.mozilla-subhub.app</li>
-    <li>subscriptions.firefox.com</li>
-  </ul>
-
   <h3>Pocket</h3>
   <ul class="mzp-u-list-styled">
     <li>getpocket.com</li>
     <li>api.getpocket.com</li>
-    <li>app.getpocket.com</li>
-    <li>widgets.getpocket.com</li>
```

Comment:
Both app.getpocket and widgets.getpocket still seem to exist, though I don't know how they're used.

Reply:
I'll leave them then

Reply:
app.getpocket.com redirects to getpocket and widgets.getpocket.com redirect to [publisher and both under getpocket](https://getpocket.com/publisher/) so we can consider them under included with getpocket

Reply:
The top-level redirects (or maybe any 404), but there could still be specific API calls from their mobile apps or web app. If the list is "examples" then it's fine to leave off and we can cover it under "Pocket". Leaving them on (if the Pocket folks say they're still used) might attract someone to poke at them--but probably not since it hasn't happened yet! if the list is exclusive, however, then leaving them off could actively discourage bughunting there.

We should double-check our Safe Harbor wording to make sure _that_ covers "related" research. It could be that we went to the "explicit and complete list" approach because of that. Marshall would likely remember.

Reply:
this is the statement in the policy, https://www.mozilla.org/en-US/security/bug-bounty/, 

>  This includes any claim under the DMCA for circumventing technological measures to protect the services and applications eligible under this policy.

don't see something about related research

Reply:
checked with the pocket team, widgets.getpocket.com is used to download the pocket button, https://getpocket.com/publisher/button